extension/content.js

```diff
@@ -12,8 +12,12 @@ const getUsername = () => $('meta[name="user-login"]').attr('content');
 function linkifyBranchRefs() {
 	$('.commit-ref').each((i, el) => {
 		const parts = $(el).find('.css-truncate-target');
-		let username = parts.eq(0).text();
-		let branch = parts.eq(1).text();
+		let branch = parts.eq(parts.length - 1).text();
+
+		// if there are two parts the first part is the username
+		if (parts.length > 1) {
+			let username = parts.eq(0).text();
+		}
```

Comment:
No, it's not the same exactly. My change will use the owner's name in the URL. The `master` branch (without this update) will use the current user's username.

Master:
![master](https://cloud.githubusercontent.com/assets/737065/13148758/a4977bf4-d62c-11e5-960b-8f7eef8f360e.png)

This branch:
![this branch](https://cloud.githubusercontent.com/assets/737065/13148764/a9406f94-d62c-11e5-86a5-35adaff81cb9.png)